admin/class-metabox.php

```diff
@@ -1000,21 +1000,19 @@ function column_content( $column_name, $post_id ) {
 		if ( $column_name === 'wpseo-score' ) {
 			$score = self::get_value( 'linkdex', $post_id );
```

Comment:
I would place the content of this if into a separate method, just to keep it readable.